QUEST_LV_0100.tsv

```diff
@@ -1798,20 +1798,20 @@ QUEST_LV_0100_20150317_001797	By the way.. I don't see that hairy creature that
 QUEST_LV_0100_20150317_001798	Okay. I received a strange report so I was wondering what happened.{nl}I heard the dead body of a monster was found near Morkos farm.
 QUEST_LV_0100_20150317_001799	That monster looked like the monster you used to have.{nl}I will give you the first mission. Search Morcus farm.{nl}Find out if it's the same breed or if it's the monster that you used to have.
 QUEST_LV_0100_20150317_001800	Good luck on your first mission.{nl}If that is him, it's so pity.
-QUEST_LV_0100_20150317_001801	Ah.. how. could.. this.. It's not true. This body must not be Spion's!!{nl}I can't believe it. Look closely, it must be a different monster.
-QUEST_LV_0100_20150317_001802	No.. no way..how could you be.. no..
-QUEST_LV_0100_20150317_001803	Ah.. Spion!! how.. because of me.. you..{nl}We are so sorry..
-QUEST_LV_0100_20150317_001804	Herb broker!! This is all because of the herb broker.{nl}I will find that broker and get revenge for Spion!{nl}Where is he by the way?
+xQUEST_LV_0100_20150317_001801	Ah... how could this... It's not true. This body must not be Spion's!!{nl}I can't believe it. Look closely, it must be a different monster.
+xQUEST_LV_0100_20150317_001802	No... No way... How could you be...{nl}No...
+xQUEST_LV_0100_20150317_001803	Ah... Spion!! How... Because of me, you...{nl}We are so sorry...
+QUEST_LV_0100_20150317_001804	Herb broker!! This is all because of the herb broker.{nl}I will find that broker and get revenge for Spion!{nl}Where is he, by the way?
 QUEST_LV_0100_20150317_001805	It's not your fault that Spion turned out like this.{nl}Even if that's your fault, it's not that Spion should die.
```

Comment:
The first sentence is fine, the second sentence could possibly go "Even if it is, he still shouldn't die."